CHANGELOG.rst

```diff
@@ -8,7 +8,15 @@ the version control of each dependency.
 4.3.0 (unreleased)
 ==================
 
-- Nothing changed yet.
+amo2kinto
+'''''''''
+
+**amo2kinto 1.7.2 → 3.0.0**: https://github.com/mozilla-services/kinto-amo/releases/tag/3.0.0
+
+**Bug fixes**
+
+- Remove json2kinto importer
+- Remove xml verifier
```

Comment:
This will lead to 5.0 right?

Reply:
Yes we could, also it doesn't change the kinto service behaviour so I don't think it is needed. amo2kinto is not a kinto plugin and the kinto-amo major didn't changed.